tests/Tests/200__monitor_and_manage/200__metrics/203__alerts.robot

```diff
@@ -0,0 +1,71 @@
+*** Settings ***
+Force Tags       Smoke  Sanity
+Resource         ../../../Resources/ODS.robot
+Resource         ../../../Resources/Common.robot
+Resource         ../../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource         ../../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library          DebugLibrary
+Library          SeleniumLibrary
+Library          JupyterLibrary
+Test Teardown   End Web Test
+
+*** Variables ***
+${rule_group}=  RHODS-PVC-Usage
+${alert_90}=  User notebook pvc usage above 90%
+${alert_100}=  User notebook pvc usage at 100%
+${notebook_repo_url}=  https://github.com/redhat-rhods-qe/ods-ci-notebooks-main
+${notebook_90}=  /ods-ci-notebooks-main/notebooks/200__monitor_and_manage/203__alerts/notebook-pvc-usage/fill-notebook-pvc-over-90.ipynb
+${notebook_100}=  /ods-ci-notebooks-main/notebooks/200__monitor_and_manage/203__alerts/notebook-pvc-usage/fill-notebook-pvc-to-100.ipynb
+${notebook_clean}=  /ods-ci-notebooks-main/notebooks/200__monitor_and_manage/203__alerts/notebook-pvc-usage/fill-notebook-pvc-delete-testfiles.ipynb
+
+*** Test Cases ***
+Verify alert RHODS-PVC-Usage is fired when user notebook pvc usage is above 90 Percent
+  [Tags]  Sanity  ODS-516 
+  Set Up Alert Test  ${notebook_90}
+  Sleep  200
+  Prometheus.Alert Should Be Firing  ${RHODS_PROMETHEUS_URL}  ${RHODS_PROMETHEUS_TOKEN}  ${rule_group}  ${alert_90}
+  Sleep  10
+
+Verify alert RHODS-PVC-Usage is fired when user notebook pvc usage is 100 Percent
+  [Tags]  Sanity  ODS-516 
+  Set Up Alert Test  ${notebook_100}
+  Sleep  200
+  Prometheus.Alert Should Be Firing  ${RHODS_PROMETHEUS_URL}  ${RHODS_PROMETHEUS_TOKEN}  ${rule_group}  ${alert_100}
+  Sleep  10
+  Execute Notebook Disk Full  ${notebook_clean}
```

Comment:
You could rename `Execute Notebook Disk Full` to `Clean Up Files And End Web Test`

Then set a custom Test teardown for this test like this (I found how to do it [here](http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#test-setup-and-teardown))
```
Verify alert RHODS-PVC-Usage is fired when user notebook pvc usage is 100 Percent
  [Tags]  Sanity  ODS-516 
  Set Up Alert Test  ${notebook_100}
  Sleep  200
  Prometheus.Alert Should Be Firing  ${RHODS_PROMETHEUS_URL}  ${RHODS_PROMETHEUS_TOKEN}  ${rule_group}  ${alert_100}
  [Teardown]  Clean Up Files And End Web Test


Clean Up Files And End Web Test
    [Documentation]  We need to clean the files here because when disk is 100% "End Web Test" cannot do it
    Maybe Accept a JupyterLab Prompt
    Sleep  10
    Add and Run JupyterLab Code Cell  !rm -rf *
    Sleep  20
    Common.End Web Test
```